--- template/app/src/client/App.tsx
+++ opensaas-sh/app/src/client/App.tsx
@@ -18,7 +18,7 @@
   const location = useLocation();
   const isMarketingPage = useMemo(() => {
     return (
-      location.pathname === "/" || location.pathname.startsWith("/pricing")
+      location.pathname === "/"
     );
   }, [location]);
 
@@ -42,7 +42,13 @@
       const id = location.hash.replace("#", "");
       const element = document.getElementById(id);
       if (element) {
-        element.scrollIntoView();
+        const yOffset = -100;
+        const top = element.getBoundingClientRect().top + window.pageYOffset + yOffset;
+        window.scrollTo({ top, behavior: "smooth" });
+      }
+    } else if (location.pathname === "/") {
+      if (window.scrollY > 0) {
+        window.scrollTo({ top: 0, behavior: "smooth" });
       }
     }
   }, [location]);
