--- template/app/src/client/App.tsx
+++ opensaas-sh/app/src/client/App.tsx
@@ -4,6 +4,7 @@
 import './Main.css';
 import NavBar from './components/NavBar/NavBar';
 import { demoNavigationitems, marketingNavigationItems } from './components/NavBar/constants';
+import { useIsLandingPage } from './hooks/useIsLandingPage';
 import CookieConsentBanner from './components/cookie-consent/Banner';
 import { Toaster } from '../components/ui/toaster';
 
@@ -13,11 +14,8 @@
  */
 export default function App() {
   const location = useLocation();
-  const isMarketingPage = useMemo(() => {
-    return location.pathname === '/' || location.pathname.startsWith('/pricing');
-  }, [location]);
-
-  const navigationItems = isMarketingPage ? marketingNavigationItems : demoNavigationitems;
+  const isLandingPage = useIsLandingPage();
+  const navigationItems = isLandingPage ? marketingNavigationItems : demoNavigationitems;
 
   const shouldDisplayAppNavBar = useMemo(() => {
     return (
