--- template/app/src/client/components/NavBar/Announcement.tsx
+++ opensaas-sh/app/src/client/components/NavBar/Announcement.tsx
@@ -1,32 +1,34 @@
-const ANNOUNCEMENT_URL = "https://github.com/wasp-lang/wasp";
+const ANNOUNCEMENT_URL = "https://x.com/hot_town/status/1950117243246821770";
 
 export function Announcement() {
+  const launchDate = new Date("2025-07-29T00:00:00-07:00");
+  const today = new Date();
+  const hasLaunched =
+    today.toISOString().slice(0, 10) >= launchDate.toISOString().slice(0, 10);
+
   return (
-    <div className="from-accent to-secondary text-primary-foreground relative flex w-full items-center justify-center gap-3 bg-gradient-to-r p-3 text-center font-semibold">
+    <div className="from-accent to-secondary text-primary-foreground relative z-[51] flex w-full items-center justify-center gap-3 bg-gradient-to-r p-3 text-center font-semibold tracking-wider">
       <a
         href={ANNOUNCEMENT_URL}
         target="_blank"
-        rel="noopener noreferrer"
-        className="hidden cursor-pointer transition-opacity hover:opacity-90 hover:drop-shadow lg:block"
+        className="hidden transition-opacity hover:opacity-90 hover:drop-shadow lg:block"
       >
-        Support Open-Source Software!
+        🚀 Open SaaS v2.0 {hasLaunched ? "is here!" : "launches tomorrow!"}
       </a>
       <div className="bg-primary-foreground/20 hidden w-0.5 self-stretch lg:block"></div>
       <a
         href={ANNOUNCEMENT_URL}
         target="_blank"
-        rel="noopener noreferrer"
-        className="bg-background/20 hover:bg-background/30 hidden cursor-pointer rounded-full px-2.5 py-1 text-xs tracking-wider transition-colors lg:block"
+        className="bg-background/20 hover:bg-background/30 hidden cursor-pointer rounded-full px-2.5 py-1 tracking-wider opacity-95 transition-colors lg:block"
       >
-        Star Our Repo on Github ⭐️ →
+        {hasLaunched ? "Check out the Launch 🎉" : "Get notified! 📆"} →
       </a>
       <a
         href={ANNOUNCEMENT_URL}
         target="_blank"
-        rel="noopener noreferrer"
         className="bg-background/20 hover:bg-background/30 cursor-pointer rounded-full px-2.5 py-1 text-xs transition-colors lg:hidden"
       >
-        ⭐️ Star the Our Repo and Support Open-Source! ⭐️
+        🎉 The Open SaaS v2.0 Launch is Live! 🚀
       </a>
     </div>
   );
