--- template/app/src/client/components/NavBar/NavBar.tsx
+++ opensaas-sh/app/src/client/components/NavBar/NavBar.tsx
@@ -10,6 +10,7 @@
 import { useIsLandingPage } from '../../hooks/useIsLandingPage';
 import logo from '../../static/logo.webp';
 import DarkModeSwitcher from '../DarkModeSwitcher';
+import RepoInfo from '../RepoInfo';
 
 export interface NavigationItem {
   name: string;
@@ -38,8 +39,12 @@
 
   return (
     <>
-      {isLandingPage && <Announcement />}
-      <header className={cn('sticky top-0 z-50 transition-all duration-300', isScrolled && 'top-4')}>
+      <header
+        className={cn(
+          'sticky top-0 z-50 transition-all duration-300',
+          isScrolled && 'top-4 mx-4 xl:mx-30 lg:mx-10'
+        )}
+      >
         <div
           className={cn('transition-all duration-300', {
             'mx-4 md:mx-20 rounded-full shadow-lg bg-background/90 backdrop-blur-lg border border-border':
@@ -67,7 +72,7 @@
                       'ml-2 text-xs': isScrolled,
                     })}
                   >
-                    Your SaaS
+                    Open SaaS
                   </span>
                 )}
               </WaspRouterLink>
@@ -99,7 +104,7 @@
                   <SheetHeader>
                     <SheetTitle className='flex items-center'>
                       <WaspRouterLink to={routes.LandingPageRoute.to}>
-                        <span className='sr-only'>Your SaaS</span>
+                        <span className='sr-only'>Open SaaS</span>
                         <NavLogo isScrolled={false} />
                       </WaspRouterLink>
                     </SheetTitle>
@@ -123,7 +128,14 @@
                         )}
                       </div>
                       <div className='py-6'>
-                        <DarkModeSwitcher />
+                        <ul className='flex items-center justify-between gap-4'>
+                          <li>
+                            <DarkModeSwitcher />
+                          </li>
+                          <li>
+                            <RepoInfo />
+                          </li>
+                        </ul>
                       </div>
                     </div>
                   </div>
@@ -132,8 +144,13 @@
             </div>
 
             <div className='hidden lg:flex lg:flex-1 gap-3 justify-end items-center'>
-              <ul className='flex justify-center items-center gap-2 sm:gap-4'>
-                <DarkModeSwitcher />
+              <ul className='flex justify-center items-center gap-2 sm:gap-6'>
+                <li>
+                  <RepoInfo />
+                </li>
+                <li>
+                  <DarkModeSwitcher />
+                </li>
               </ul>
               {isUserLoading ? null : !user ? (
                 <WaspRouterLink
@@ -246,34 +263,6 @@
       'size-6': isScrolled,
     })}
     src={logo}
-    alt='Your SaaS App'
+    alt='Open SaaS App'
   />
 );
-
-const ContestURL = 'https://github.com/wasp-lang/wasp';
-
-function Announcement() {
-  return (
-    <div className='flex justify-center items-center gap-3 p-3 w-full bg-gradient-to-r from-accent to-secondary font-semibold text-primary-foreground text-center z-49'>
-      <p
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='hidden lg:block cursor-pointer hover:opacity-90 hover:drop-shadow transition-opacity'
-      >
-        Support Open-Source Software!
-      </p>
-      <div className='hidden lg:block self-stretch w-0.5 bg-primary-foreground/20'></div>
-      <div
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='hidden lg:block cursor-pointer rounded-full bg-background/20 px-2.5 py-1 text-xs hover:bg-background/30 transition-colors tracking-wider'
-      >
-        Star Our Repo on Github ⭐️ →
-      </div>
-      <div
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='lg:hidden cursor-pointer rounded-full bg-background/20 px-2.5 py-1 text-xs hover:bg-background/30 transition-colors'
-      >
-        ⭐️ Star the Our Repo and Support Open-Source! ⭐️
-      </div>
-    </div>
-  );
-}
