--- template/app/src/client/components/NavBar/NavBar.tsx
+++ opensaas-sh/app/src/client/components/NavBar/NavBar.tsx
@@ -38,11 +38,15 @@
 
   return (
     <>
-      {isLandingPage && <Announcement />}
-      <header className={cn('sticky top-0 z-50 transition-all duration-300', isScrolled && 'top-4')}>
+      <header
+        className={cn(
+          'sticky top-0 z-50 transition-all duration-300',
+          isScrolled && 'top-4 mx-4 xl:mx-30 lg:mx-10'
+        )}
+      >
         <div
           className={cn('transition-all duration-300', {
-            'mx-4 md:mx-20 rounded-full shadow-lg bg-background/90 backdrop-blur-lg border border-border':
+            'mx-auto rounded-full shadow-outer dark:shadow bg-background/90 backdrop-blur-lg border border-border':
               isScrolled,
             'mx-0 bg-background/80 backdrop-blur-lg border-b border-border': !isScrolled,
           })}
@@ -67,7 +71,7 @@
                       'ml-2 text-xs': isScrolled,
                     })}
                   >
-                    Your SaaS
+                    Open SaaS
                   </span>
                 )}
               </WaspRouterLink>
@@ -99,7 +103,7 @@
                   <SheetHeader>
                     <SheetTitle className='flex items-center'>
                       <WaspRouterLink to={routes.LandingPageRoute.to}>
-                        <span className='sr-only'>Your SaaS</span>
+                        <span className='sr-only'>Open SaaS</span>
                         <NavLogo isScrolled={false} />
                       </WaspRouterLink>
                     </SheetTitle>
@@ -246,34 +250,6 @@
       'size-6': isScrolled,
     })}
     src={logo}
-    alt='Your SaaS App'
+    alt='Open SaaS App'
   />
 );
-
-const ContestURL = 'https://github.com/wasp-lang/wasp';
-
-function Announcement() {
-  return (
-    <div className='flex justify-center items-center gap-3 p-3 w-full bg-gradient-to-r from-accent to-secondary font-semibold text-primary-foreground text-center z-49'>
-      <p
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='hidden lg:block cursor-pointer hover:opacity-90 hover:drop-shadow transition-opacity'
-      >
-        Support Open-Source Software!
-      </p>
-      <div className='hidden lg:block self-stretch w-0.5 bg-primary-foreground/20'></div>
-      <div
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='hidden lg:block cursor-pointer rounded-full bg-background/20 px-2.5 py-1 text-xs hover:bg-background/30 transition-colors tracking-wider'
-      >
-        Star Our Repo on Github ⭐️ →
-      </div>
-      <div
-        onClick={() => window.open(ContestURL, '_blank')}
-        className='lg:hidden cursor-pointer rounded-full bg-background/20 px-2.5 py-1 text-xs hover:bg-background/30 transition-colors'
-      >
-        ⭐️ Star the Our Repo and Support Open-Source! ⭐️
-      </div>
-    </div>
-  );
-}
