--- template/app/src/file-upload/file-upload.wasp.ts
+++ opensaas-sh/app/src/file-upload/file-upload.wasp.ts
@@ -1,4 +1,4 @@
-import { action, page, query, route, type Part } from "@wasp.sh/spec";
+import { action, job, page, query, route, type Part } from "@wasp.sh/spec";
 
 import { group } from "../shared/utils.wasp";
 import { FileUploadPage } from "./FileUploadPage" with { type: "ref" };
@@ -9,6 +9,7 @@
   getAllFilesByUser,
   getDownloadFileSignedURL,
 } from "./operations" with { type: "ref" };
+import { deleteFilesJob } from "./workers" with { type: "ref" };
 
 export const fileUpload: Part[] = [
   route(
@@ -24,4 +25,11 @@
     action(addFileToDb),
     action(deleteFile),
   ]),
+  job(deleteFilesJob, {
+    executor: "PgBoss",
+    schedule: {
+      cron: "0 5 * * *", // every day at 5am
+    },
+    entities: ["File"],
+  }),
 ];
