--- template/app/src/file-upload/operations.ts
+++ opensaas-sh/app/src/file-upload/operations.ts
@@ -37,6 +37,18 @@
     userId: context.user.id,
   });
 
+  const numberOfFilesByUser = await context.entities.File.count({
+    where: {
+      user: {
+        id: context.user.id,
+      },
+    },
+  });
+
+  if (numberOfFilesByUser >= 2) {
+    throw new HttpError(403, 'Thanks for trying Open SaaS. This demo only allows 2 file uploads per user.');
+  }
+
   await context.entities.File.create({
     data: {
       name: fileName,
