--- template/app/src/admin/layout/Header.tsx
+++ opensaas-sh/app/src/admin/layout/Header.tsx
@@ -27,7 +27,7 @@
               <span className='du-block absolute right-0 h-full w-full'>
                 <span
                   className={cn(
-                    'relative top-0 left-0 my-1 block h-0.5 w-0 rounded-sm bg-foreground delay-[0] duration-200 ease-in-out',
+                    'relative top-0 left-0 my-1 block h-0.5 w-0 rounded-sm bg-foreground delay-0 duration-200 ease-in-out',
                     {
                       '!w-full delay-300': !props.sidebarOpen,
                     }
@@ -55,7 +55,7 @@
                   className={cn(
                     'absolute left-2.5 top-0 block h-full w-0.5 rounded-sm bg-foreground delay-300 duration-200 ease-in-out',
                     {
-                      '!h-0 !delay-[0]': !props.sidebarOpen,
+                      '!h-0 !delay-0': !props.sidebarOpen,
                     }
                   )}
                 ></span>
