--- template/app/src/payment/paymentProcessor.ts
+++ opensaas-sh/app/src/payment/paymentProcessor.ts
@@ -3,7 +3,6 @@
 import type { MiddlewareConfigFn } from 'wasp/server';
 import { PrismaClient } from '@prisma/client';
 import { stripePaymentProcessor } from './stripe/paymentProcessor';
-import { lemonSqueezyPaymentProcessor } from './lemonSqueezy/paymentProcessor';
 
 export interface CreateCheckoutSessionArgs {
   userId: string;
@@ -24,9 +23,4 @@
   webhookMiddlewareConfigFn: MiddlewareConfigFn;
 }
 
-/**
- * Choose which payment processor you'd like to use, then delete the 
- * other payment processor code that you're not using  from `/src/payment`
- */
-// export const paymentProcessor: PaymentProcessor = lemonSqueezyPaymentProcessor;
 export const paymentProcessor: PaymentProcessor = stripePaymentProcessor;
