--- template/app/src/payment/paymentProcessor.ts
+++ opensaas-sh/app/src/payment/paymentProcessor.ts
@@ -27,9 +27,4 @@
   webhookMiddlewareConfigFn: MiddlewareConfigFn;
 }
 
-/**
- * Choose which payment processor you'd like to use, then delete the
- * other payment processor code that you're not using  from `/src/payment`
- */
-// export const paymentProcessor: PaymentProcessor = lemonSqueezyPaymentProcessor;
 export const paymentProcessor: PaymentProcessor = stripePaymentProcessor;
