--- template/app/src/user/AccountPage.tsx
+++ opensaas-sh/app/src/user/AccountPage.tsx
@@ -33,7 +33,6 @@
                 subscriptionPlan={user.subscriptionPlan}
                 datePaid={user.datePaid}
                 credits={user.credits}
-                lemonSqueezyCustomerPortalUrl={user.lemonSqueezyCustomerPortalUrl}
               />
             </div>
             <div className='py-4 sm:grid sm:grid-cols-3 sm:gap-4 sm:py-5 sm:px-6'>
@@ -60,10 +59,9 @@
   subscriptionStatus: SubscriptionStatus | null;
   datePaid: Date | null;
   credits: number;
-  lemonSqueezyCustomerPortalUrl: string | null;
 };
 
-function UserCurrentPaymentPlan({ subscriptionPlan, subscriptionStatus, datePaid, credits, lemonSqueezyCustomerPortalUrl }: UserCurrentPaymentPlanProps) {
+function UserCurrentPaymentPlan({ subscriptionPlan, subscriptionStatus, datePaid, credits }: UserCurrentPaymentPlanProps) {
   if (subscriptionStatus && subscriptionPlan && datePaid) {
     return (
       <>
