--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -41,7 +41,10 @@
 };
 
 type GetPaginatedUsersOutput = {
-  users: Pick<User, 'id' | 'email' | 'username' | 'subscriptionStatus' | 'paymentProcessorUserId'>[];
+  users: Pick<
+    User,
+    'id' | 'email' | 'username' | 'subscriptionStatus' | 'stripeId'
+  >[];
   totalPages: number;
 };
 
@@ -82,6 +85,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
@@ -105,7 +109,7 @@
       username: true,
       isAdmin: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
     orderBy: {
       id: 'desc',
@@ -121,6 +125,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
