--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -1,9 +1,9 @@
-import * as z from 'zod';
-import { type UpdateIsUserAdminById, type GetPaginatedUsers } from 'wasp/server/operations';
+import { type Prisma } from '@prisma/client';
 import { type User } from 'wasp/entities';
 import { HttpError, prisma } from 'wasp/server';
+import { type GetPaginatedUsers, type UpdateIsUserAdminById } from 'wasp/server/operations';
+import * as z from 'zod';
 import { SubscriptionStatus } from '../payment/plans';
-import { type Prisma } from '@prisma/client';
 import { ensureArgsSchemaOrThrowHttpError } from '../server/validation';
 
 const updateUserAdminByIdInputSchema = z.object({
@@ -34,10 +34,7 @@
 };
 
 type GetPaginatedUsersOutput = {
-  users: Pick<
-    User,
-    'id' | 'email' | 'username' | 'subscriptionStatus' | 'paymentProcessorUserId' | 'isAdmin'
-  >[];
+  users: Pick<User, 'id' | 'email' | 'username' | 'subscriptionStatus' | 'stripeId' | 'isAdmin'>[];
   totalPages: number;
 };
 
@@ -85,6 +82,7 @@
             mode: 'insensitive',
           },
           isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
@@ -106,7 +104,7 @@
       username: true,
       isAdmin: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
     orderBy: {
       username: 'asc',
