--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -52,7 +52,10 @@
   subscriptionStatus?: SubscriptionStatus[];
 };
 type GetPaginatedUsersOutput = {
-  users: Pick<User, 'id' | 'email' | 'username' | 'lastActiveTimestamp' | 'subscriptionStatus' | 'paymentProcessorUserId'>[];
+  users: Pick<
+    User,
+    'id' | 'email' | 'username' | 'lastActiveTimestamp' | 'subscriptionStatus' | 'stripeId'
+  >[];
   totalPages: number;
 };
 
@@ -65,8 +68,10 @@
   }
 
   const allSubscriptionStatusOptions = args.subscriptionStatus as Array<string | null> | undefined;
-  const hasNotSubscribed = allSubscriptionStatusOptions?.find((status) => status === null) 
-  let subscriptionStatusStrings = allSubscriptionStatusOptions?.filter((status) => status !== null) as string[] | undefined
+  const hasNotSubscribed = allSubscriptionStatusOptions?.find((status) => status === null);
+  let subscriptionStatusStrings = allSubscriptionStatusOptions?.filter((status) => status !== null) as
+    | string[]
+    | undefined;
 
   const queryResults = await context.entities.User.findMany({
     skip: args.skip,
@@ -79,6 +84,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
@@ -103,7 +109,7 @@
       isAdmin: true,
       lastActiveTimestamp: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
     orderBy: {
       id: 'desc',
@@ -119,6 +125,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
