--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -2,7 +2,7 @@
 import { type UpdateIsUserAdminById, type GetPaginatedUsers } from 'wasp/server/operations';
 import { type User } from 'wasp/entities';
 import { HttpError } from 'wasp/server';
-import { subscriptionStatusSchema, type SubscriptionStatus } from '../payment/plans';
+import { subscriptionStatusSchema } from '../payment/plans';
 import { ensureArgsSchemaOrThrowHttpError } from '../server/validation';
 
 const updateUserAdminByIdInputSchema = z.object({
@@ -33,10 +33,7 @@
 };
 
 type GetPaginatedUsersOutput = {
-  users: Pick<
-    User,
-    'id' | 'email' | 'username' | 'subscriptionStatus' | 'paymentProcessorUserId' | 'isAdmin'
-  >[];
+  users: Pick<User, 'id' | 'email' | 'username' | 'subscriptionStatus' | 'stripeId' | 'isAdmin'>[];
   totalPages: number;
 };
 
@@ -80,6 +77,7 @@
             mode: 'insensitive',
           },
           isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
@@ -103,7 +101,7 @@
       username: true,
       isAdmin: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
     orderBy: {
       id: 'desc',
@@ -119,6 +117,7 @@
             mode: 'insensitive',
           },
           isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
