--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -34,10 +34,7 @@
 };
 
 type GetPaginatedUsersOutput = {
-  users: Pick<
-    User,
-    'id' | 'email' | 'username' | 'subscriptionStatus' | 'paymentProcessorUserId' | 'isAdmin'
-  >[];
+  users: Pick<User, 'id' | 'email' | 'username' | 'subscriptionStatus' | 'stripeId' | 'isAdmin'>[];
   totalPages: number;
 };
 
@@ -85,6 +82,7 @@
             mode: 'insensitive',
           },
           isAdmin,
+          isMockUser: true,
         },
         {
           OR: [
@@ -106,7 +104,7 @@
       username: true,
       isAdmin: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
     orderBy: {
       username: 'asc',
