--- template/app/src/user/operations.ts
+++ opensaas-sh/app/src/user/operations.ts
@@ -50,7 +50,7 @@
   subscriptionStatus?: SubscriptionStatus[];
 };
 type GetPaginatedUsersOutput = {
-  users: Pick<User, 'id' | 'email' | 'username' | 'lastActiveTimestamp' | 'subscriptionStatus' | 'paymentProcessorUserId'>[];
+  users: Pick<User, 'id' | 'email' | 'username' | 'lastActiveTimestamp' | 'subscriptionStatus' | 'stripeId' | 'lemonSqueezyId'>[];
   totalPages: number;
 };
 
@@ -77,6 +77,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true
         },
         {
           OR: [
@@ -101,7 +102,8 @@
       isAdmin: true,
       lastActiveTimestamp: true,
       subscriptionStatus: true,
-      paymentProcessorUserId: true,
+      stripeId: true,
+      lemonSqueezyId: true
     },
     orderBy: {
       id: 'desc',
@@ -117,6 +119,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true
         },
         {
           OR: [
