--- template/app/src/env.ts
+++ opensaas-sh/app/src/env.ts
@@ -2,8 +2,6 @@
 
 import { authEnvSchema } from './auth/env'
 import { stripeEnvSchema } from './payment/stripe/env'
-import { lemonSqueezyEnvSchema } from './payment/lemonSqueezy/env'
-import { polarEnvSchema } from './payment/polar/env'
 import { demoAiAppEnvSchema } from './demo-ai-app/env'
 import { fileUploadEnvSchema } from './file-upload/env'
 import { plausibleEnvSchema, googleAnalyticsEnvSchema } from './analytics/env'
@@ -18,8 +16,6 @@
 export const serverEnvValidationSchema = defineEnvValidationSchema(
   authEnvSchema
     .merge(stripeEnvSchema)
-    .merge(lemonSqueezyEnvSchema)
-    .merge(polarEnvSchema)
     .merge(demoAiAppEnvSchema)
     .merge(fileUploadEnvSchema)
     .merge(plausibleEnvSchema)
