--- template/app/src/landing-page/components/Footer.tsx
+++ opensaas-sh/app/src/landing-page/components/Footer.tsx
@@ -13,7 +13,7 @@
     <div className='mx-auto mt-6 max-w-7xl px-6 lg:px-8 dark:bg-boxdark-2'>
       <footer
         aria-labelledby='footer-heading'
-        className='relative border-t border-gray-900/10 dark:border-gray-200/10 py-24 sm:mt-32'
+        className='relative border-t border-gray-900/10 dark:border-gray-100/10 py-24 sm:mt-32'
       >
         <h2 id='footer-heading' className='sr-only'>
           Footer
