--- template/app/src/landing-page/components/Hero.tsx
+++ opensaas-sh/app/src/landing-page/components/Hero.tsx
@@ -1,7 +1,25 @@
-import openSaasBannerWebp from '../../client/static/open-saas-banner.webp';
-import { DocsUrl } from '../../shared/common';
+import { useState, useEffect } from 'react';
+import { AiFillGithub } from 'react-icons/ai';
+import { DocsUrl, GithubUrl } from '../../shared/common';
 
 export default function Hero() {
+  const [repoInfo, setRepoInfo] = useState<null | any>(null);
+
+  useEffect(() => {
+    const fetchRepoInfo = async () => {
+      try {
+        const response = await fetch(
+          'https://api.github.com/repos/wasp-lang/open-saas'
+        );
+        const data = await response.json();
+        setRepoInfo(data);
+      } catch (error) {
+        console.error('Error fetching repo info', error);
+      }
+    };
+    fetchRepoInfo();
+  }, []);
+
   return (
     <div className='relative pt-14 w-full'>
       <TopGradient />
@@ -9,31 +27,47 @@
       <div className='py-24 sm:py-32'>
         <div className='mx-auto max-w-8xl px-6 lg:px-8'>
           <div className='lg:mb-18 mx-auto max-w-3xl text-center'>
-            <h1 className='text-4xl font-bold text-gray-900 sm:text-6xl dark:text-white'>
-              Some <span className='italic'>cool</span> words about your product
+            <h1 className='text-4xl font-bold text-gray-900 sm:text-6xl dark:text-white tracking-tight'>
+              The <span className='italic'>free</span> SaaS template with
+              superpowers
             </h1>
-            <p className='mt-6 mx-auto max-w-2xl text-lg leading-8 text-gray-600 dark:text-white'>
-              With some more exciting words about your product!
+            <p className='mt-6 mx-auto max-w-2xl text-lg leading-8 text-gray-600 dark:text-gray-400'>
+              An open-source, feature-rich, full-stack React + NodeJS
+              template that manages features for you.
             </p>
             <div className='mt-10 flex items-center justify-center gap-x-6'>
               <a
                 href={DocsUrl}
-                className='rounded-md px-3.5 py-2.5 text-sm font-semibold text-gray-700 ring-1 ring-inset ring-gray-200 hover:ring-2 hover:ring-yellow-300 shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 dark:text-white'
+                className='rounded-md px-6 py-4 text-sm font-semibold text-gray-700 ring-1 ring-inset ring-gray-200 hover:ring-2 hover:ring-yellow-300 shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600 dark:text-white'
               >
                 Get Started <span aria-hidden='true'>→</span>
               </a>
+              <a
+                href={GithubUrl}
+                className='group relative flex items-center justify-center rounded-md bg-gray-100 px-6 py-4 text-sm font-semibold shadow-sm ring-1 ring-inset ring-gray-200 dark:bg-gray-700 hover:ring-2 hover:ring-yellow-300 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600'
+              >
+                {/* <AiFillGithub size='1.25rem' className='mr-2' /> */}
+                View the Repo
+                {repoInfo!! && (
+                  <>
+                    <span className='absolute -top-3 -right-7 inline-flex items-center gap-x-1 rounded-full ring-1 group-hover:ring-2 ring-inset ring-yellow-300 bg-yellow-100 px-2 py-1 text-sm font-medium text-yellow-800'>
+                      <AiFillGithub size='1rem' />
+                      {repoInfo.stargazers_count}
+                    </span>
+                  </>
+                )}
+              </a>
             </div>
           </div>
-          <div className='mt-14 flow-root sm:mt-14'>
-            <div className='-m-2  flex justify-center rounded-xl lg:-m-4 lg:rounded-2xl lg:p-4'>
-              <img
-                src={openSaasBannerWebp}
-                alt='App screenshot'
-                width={1000}
-                height={530}
-                loading='lazy'
-                className='rounded-md shadow-2xl ring-1 ring-gray-900/10'
-              />
+          <div className='mt-14 flow-root sm:mt-14 '>
+            <div className='-m-2 mx-auto rounded-xl lg:-m-4 lg:rounded-2xl lg:p-4'>
+              <iframe
+                className=' mx-auto w-full md:w-[85%] aspect-[4/3] shadow-2xl'
+                src='https://cards.producthunt.com/cards/posts/436467?v=1'
+                // width={850}
+                // height={689}
+                allowFullScreen
+              ></iframe>
             </div>
           </div>
         </div>
