--- template/app/src/landing-page/components/HighlightedFeature.tsx
+++ opensaas-sh/app/src/landing-page/components/HighlightedFeature.tsx
@@ -42,7 +42,7 @@
         direction === 'row' ? 'md:flex-row' : 'md:flex-row-reverse'
       )}
     >
-      <div className='flex-col flex-1 w-full'>
+      <div className='flex-col flex-1'>
         <h2 className='text-4xl font-bold mb-2'>{name}</h2>
         {typeof description === 'string' ? (
           <p className='text-muted-foreground'>{description}</p>
