--- template/app/src/admin/dashboards/users/UsersDashboardPage.tsx
+++ opensaas-sh/app/src/admin/dashboards/users/UsersDashboardPage.tsx
@@ -1,14 +1,50 @@
 import { type AuthUser } from 'wasp/auth';
+import { useState, useEffect } from 'react';
 import UsersTable from './UsersTable';
 import Breadcrumb from '../../layout/Breadcrumb';
 import DefaultLayout from '../../layout/DefaultLayout';
 import { useRedirectHomeUnlessUserIsAdmin } from '../../useRedirectHomeUnlessUserIsAdmin';
 
 const Users = ({ user }: { user: AuthUser }) => {
-  useRedirectHomeUnlessUserIsAdmin({user})
+  const [isDemoInfoVisible, setIsDemoInfoVisible] = useState(false);
+  useRedirectHomeUnlessUserIsAdmin({user});
+
+  useEffect(() => {
+    try {
+      if (localStorage.getItem('isDemoInfoVisible') === 'false') {
+        // do nothing
+      } else {
+        setIsDemoInfoVisible(true);
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  }, []);
+
+  const handleDemoInfoClose = () => {
+    try {
+      localStorage.setItem('isDemoInfoVisible', 'false');
+      setIsDemoInfoVisible(false);
+    } catch (error) {
+      console.error(error);
+    }
+  };
 
   return (
     <DefaultLayout user={user}>
+      {/* Floating Demo Announcement */}
+      {isDemoInfoVisible && (
+        <div className='fixed z-999 bottom-0 mb-2 left-1/2 -translate-x-1/2 lg:mb-4 bg-gray-700 rounded-full px-3.5 py-2 text-sm text-white duration-300 ease-in-out hover:bg-gray-800 focus-visible:outline focus-visible:outline-2 focus-visible:outline-indigo-600'>
+          <div className='px-4 flex flex-row gap-2 items-center my-1'>
+            <span className='text-gray-100'>
+              You are viewing mock user data only ;)
+            </span>
+            <button className=' pl-2.5 text-gray-400 text-xl font-bold' onClick={() => handleDemoInfoClose()}>
+              X
+            </button>
+          </div>
+        </div>
+      )}
       <Breadcrumb pageName='Users' />
       <div className='flex flex-col gap-10'>
         <UsersTable />
