--- template/app/src/admin/dashboards/users/UsersTable.tsx
+++ opensaas-sh/app/src/admin/dashboards/users/UsersTable.tsx
@@ -254,7 +254,7 @@
                 <p className='text-sm text-foreground'>{user.subscriptionStatus}</p>
               </div>
               <div className='col-span-2 flex items-center'>
-                <p className='text-sm text-muted-foreground'>{user.paymentProcessorUserId}</p>
+                <p className='text-sm text-muted-foreground'>{user.subscriptionStatus}</p>
               </div>
               <div className='col-span-1 flex items-center'>
                 <div className='text-sm text-foreground'>
