--- template/app/src/admin/dashboards/users/UsersTable.tsx
+++ opensaas-sh/app/src/admin/dashboards/users/UsersTable.tsx
@@ -9,6 +9,7 @@
   const [skip, setskip] = useState(0);
   const [page, setPage] = useState(1);
   const [email, setEmail] = useState<string | undefined>(undefined);
+  
   const [isAdminFilter, setIsAdminFilter] = useState<boolean | undefined>(undefined);
   const [statusOptions, setStatusOptions] = useState<SubscriptionStatus[]>([]);
   const { data, isLoading, error } = useQuery(getPaginatedUsers, {
