--- template/app/src/admin/dashboards/users/UsersTable.tsx
+++ opensaas-sh/app/src/admin/dashboards/users/UsersTable.tsx
@@ -9,6 +9,7 @@
   const [skip, setskip] = useState(0);
   const [page, setPage] = useState(1);
   const [email, setEmail] = useState<string | undefined>(undefined);
+  
   const [isAdminFilter, setIsAdminFilter] = useState<boolean | undefined>(undefined);
   const [statusOptions, setStatusOptions] = useState<SubscriptionStatus[]>([]);
   const { data, isLoading, error } = useQuery(getPaginatedUsers, {
@@ -183,7 +184,7 @@
             <p className='font-medium'>Subscription Status</p>
           </div>
           <div className='col-span-2 flex items-center'>
-            <p className='font-medium'>Stripe ID</p>
+            <p className='font-medium'>Stripe / LemonSqueezy ID</p>
           </div>
           <div className='col-span-1 flex items-center'>
             <p className='font-medium'>Is Admin</p>
@@ -222,7 +223,7 @@
                 <p className='text-sm text-black dark:text-white'>{user.subscriptionStatus}</p>
               </div>
               <div className='col-span-2 flex items-center'>
-                <p className='text-sm text-meta-3'>{user.paymentProcessorUserId}</p>
+                <p className='text-sm text-meta-3'>{user.stripeId || user.lemonSqueezyId}</p>
               </div>
               <div className='col-span-1 flex items-center'>
                 <div className='text-sm text-black dark:text-white'>
