--- template/app/src/auth/auth.wasp.ts
+++ opensaas-sh/app/src/auth/auth.wasp.ts
@@ -32,6 +32,9 @@
     // NOTE: If you decide to not use email auth, make sure to also delete the related routes below.
     //       (RequestPasswordResetRoute, PasswordResetRoute, EmailVerificationRoute)
     email: getEmailAuthMethod(),
+    google: getGoogleAuthMethod(),
+    gitHub: getGitGubAuthMethod(),
+    discord: getDiscordAuthMethod(),
   },
   onAuthFailedRedirectTo: "/login",
   onAuthSucceededRedirectTo: "/demo-app",
@@ -57,7 +60,7 @@
   return {
     fromField: {
       name: "Open SaaS App",
-      email: "me@example.com",
+      email: "noreply@mg.wasp.sh",
     },
     emailVerification: {
       clientRoute: "EmailVerificationRoute",
