--- template/app/src/auth/LoginPage.tsx
+++ opensaas-sh/app/src/auth/LoginPage.tsx
@@ -1,8 +1,15 @@
+import { Navigate } from 'react-router-dom';
 import { Link as WaspRouterLink, routes } from 'wasp/client/router';
-import { LoginForm } from 'wasp/client/auth';
+import { LoginForm, useAuth } from 'wasp/client/auth';
 import { AuthPageLayout } from './AuthPageLayout';
 
 export default function Login() {
+  const { data: user } = useAuth();
+
+  if (user) {
+    return <Navigate to='/demo-app' />;
+  }
+
   return (
     <AuthPageLayout>
       <LoginForm />
