--- template/app/src/auth/userSignupFields.ts
+++ opensaas-sh/app/src/auth/userSignupFields.ts
@@ -1,11 +1,8 @@
 import { z } from 'zod';
 import { defineUserSignupFields } from 'wasp/auth/providers/types';
 
-const adminEmails = process.env.ADMIN_EMAILS?.split(',') || [];
-
 export const getEmailUserFields = defineUserSignupFields({
   username: (data: any) => data.email,
-  isAdmin: (data: any) => adminEmails.includes(data.email),
   email: (data: any) => data.email,
 });
 
@@ -29,10 +26,6 @@
     const githubData = githubDataSchema.parse(data);
     return githubData.profile.login;
   },
-  isAdmin: (data) => {
-    const githubData = githubDataSchema.parse(data);
-    return adminEmails.includes(githubData.profile.emails[0].email);
-  },
 });
 
 // NOTE: if we don't want to access users' emails, we can use scope ["user:read"]
@@ -58,10 +51,6 @@
     const googleData = googleDataSchema.parse(data);
     return googleData.profile.email;
   },
-  isAdmin: (data) => {
-    const googleData = googleDataSchema.parse(data);
-    return adminEmails.includes(googleData.profile.email);
-  },
 });
 
 export function getGoogleAuthConfig() {
@@ -86,10 +75,6 @@
     const discordData = discordDataSchema.parse(data);
     return discordData.profile.username;
   },
-  isAdmin: (data) => {
-    const email = discordDataSchema.parse(data).profile.email;
-    return !!email && adminEmails.includes(email);
-  },
 });
 
 export function getDiscordAuthConfig() {
