--- template/app/src/client/components/cookie-consent/Config.ts
+++ opensaas-sh/app/src/client/components/cookie-consent/Config.ts
@@ -3,6 +3,7 @@
 declare global {
   interface Window {
     dataLayer: any;
+    Reo: any;
   }
 }
 
@@ -15,8 +16,9 @@
     disablePageInteraction: false,
     hideFromBots: import.meta.env.PROD ? true : false, // Set this to false for dev/headless tests otherwise the modal will not be visible.
     mode: "opt-in",
-    // Bump the revision field when you add new services
-    revision: 0,
+    // Bump the revision field when you add new services.
+    // Keep this in sync with: opensaas-sh/blog/src/components/CookieConsentBanner.astro
+    revision: 1,
 
     // Default configuration for the cookie.
     cookie: {
@@ -50,6 +52,9 @@
             {
               name: "_gid", // string: exact cookie name
             },
+            {
+              name: /^__sec__/, // regex: match all REO.dev cookies (https://docs.reo.dev/integrations/tracking-beacon/reo.dev-javascript-cookies-and-consent-guide#key-features-of-reo.dev-cookies)
+            },
           ],
         },
 
@@ -82,6 +87,24 @@
             },
             onReject: () => {},
           },
+          reo: {
+            label: "REO.dev",
+            onAccept: () => {
+              try {
+                const REO_CLIENT_ID = "96f303a127451b4";
+                const script = document.createElement("script");
+                script.src = `https://static.reo.dev/${REO_CLIENT_ID}/reo.js`;
+                script.defer = true;
+                script.onload = () => {
+                  window.Reo.init({ clientID: REO_CLIENT_ID });
+                };
+                document.head.appendChild(script);
+              } catch (error) {
+                console.error(error);
+              }
+            },
+            onReject: () => {},
+          },
         },
       },
     },
@@ -98,10 +121,7 @@
             acceptNecessaryBtn: "Reject all",
             // showPreferencesBtn: 'Manage Individual preferences', // (OPTIONAL) Activates the preferences modal
             // TODO: Add your own privacy policy and terms and conditions links below.
-            footer: `
-            <a href="<your-url-here>" target="_blank">Privacy Policy</a>
-            <a href="<your-url-here>" target="_blank">Terms and Conditions</a>
-                    `,
+            footer: `<a href="https://docs.opensaas.sh/general/privacy-policy" target="_blank">Privacy Policy</a>`,
           },
           // The showPreferencesBtn activates this modal to manage individual preferences https://cookieconsent.orestbida.com/reference/configuration-reference.html#translation-preferencesmodal
           preferencesModal: {
