--- template/app/schema.prisma
+++ opensaas-sh/app/schema.prisma
@@ -14,9 +14,13 @@
   email                     String?         @unique
   username                  String?         @unique
   lastActiveTimestamp       DateTime        @default(now())
-  isAdmin                   Boolean         @default(false)
+  isAdmin                   Boolean         @default(true)
+  // isMockUser is an extra property for the demo app ensuring that all users can access
+  // the admin dashboard but won't be able to see the other users' data, only mock user data.
+  isMockUser                Boolean         @default(false)
 
-  paymentProcessorUserId    String?         @unique
+  stripeId                  String?         @unique
+  lemonSqueezyId            String?         @unique
   lemonSqueezyCustomerPortalUrl String?     // You can delete this if you're not using Lemon Squeezy as your payments processor.
   checkoutSessionId         String?
   subscriptionStatus        String?         // 'active', 'canceled', 'past_due', 'deleted'
