--- template/app/src/landing-page/contentSections.tsx
+++ opensaas-sh/app/src/landing-page/contentSections.tsx
@@ -0,0 +1,211 @@
+import { routes } from 'wasp/client/router';
+import type { NavigationItem } from '../client/components/NavBar/NavBar';
+import logo from '../client/static/logo.webp';
+import openSaasBannerWebp from '../client/static/open-saas-banner.webp';
+import { BlogUrl, DocsUrl, GithubUrl } from '../shared/common';
+
+export const landingPageNavigationItems: NavigationItem[] = [
+  { name: 'Features', to: '#features' },
+  { name: 'Documentation', to: DocsUrl },
+  { name: 'Blog', to: BlogUrl },
+];
+export const features = [
+  {
+    name: 'Deploy Anywhere. Easily.',
+    description:
+      'No vendor lock-in because you own all your code. Deploy yourself, or let Wasp deploy it for you with a single command.',
+    icon: '🚀 ',
+    href: DocsUrl + '/guides/deploying/',
+  },
+  {
+    name: 'File Uploads with AWS',
+    description: 'File upload examples with AWS S3 presigned URLs are included and fully documented!',
+    icon: '📁',
+    href: DocsUrl + '/guides/file-uploading/',
+  },
+  {
+    name: 'Email Sending',
+    description:
+      'Email sending built-in. Combine it with the cron jobs feature to easily send emails to your customers.',
+    icon: '📧',
+    href: DocsUrl + '/guides/email-sending/',
+    span: 2,
+  },
+  {
+    name: 'Open-Source Philosophy',
+    description: 'The repo and framework are 100% open-source',
+    icon: '🤝',
+    href: DocsUrl,
+    span: 0.5,
+  },
+  {
+    name: 'Full-stack Type Safety',
+    description: 'TypeScript with auto-generated types',
+    icon: '🥞',
+    href: DocsUrl,
+    span: 0.5,
+  },
+  {
+    name: 'Open SaaS',
+    description: 'Try the demo app',
+    icon: <img src={logo} alt='Wasp Logo' className='w-8 h-8' />,
+    href: routes.LoginRoute.to,
+    span: 1,
+  },
+  {
+    name: 'OpenAI API Implemented',
+    description: 'Have a sweet AI-powered app concept? Get your idea shipped to potential customers in days!',
+    icon: '🤖',
+    href: DocsUrl,
+  },
+  {
+    name: 'Complete Documentation & Support',
+    description: 'And a Discord community to help!',
+    icon: '🫂',
+    href: DocsUrl,
+    span: 0.5,
+  },
+  {
+    name: 'E2E Tests w/ Playwright',
+    description: 'Tests and a CI pipeline w/ GitHub Actions',
+    icon: '🧪',
+    href: DocsUrl + '/guides/tests/',
+    span: 0.5,
+  },
+  {
+    name: 'Blog w/ Astro',
+    description:
+      'Built-in blog with the Astro framework. Write your posts in Markdown, and watch your SEO performance take off.',
+    icon: '📝',
+    href: DocsUrl + '/start/guided-tour/',
+    span: 1,
+  },
+];
+export const testimonials = [
+  {
+    name: 'Max Khamrovskyi',
+    role: 'Senior Eng @ Red Hat',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1719397191205179392/V_QrGPSO_400x400.jpg',
+    socialUrl: 'https://twitter.com/maksim36ua',
+    quote:
+      'I used Wasp to build and sell my AI-augmented SaaS app for marketplace vendors within two months!',
+  },
+  {
+    name: 'Jonathan Cocharan',
+    role: 'Entrepreneur',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1910056203863883776/jtfVWaEG_400x400.jpg',
+    socialUrl: 'https://twitter.com/JonathanCochran',
+    quote:
+      'In just 6 nights... my SaaS app is live 🎉! Huge thanks to the amazing @wasplang community 🙌 for their guidance along the way. These tools are incredibly efficient 🤯!',
+  },
+  {
+    name: 'Billy Howell',
+    role: 'Entrepreneur',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1877734205561430016/jjpG4mS6_400x400.jpg',
+    socialUrl: 'https://twitter.com/billyjhowell',
+    quote:
+      "Congrats! I am loving Wasp. It's really helped me, a self-taught coder increase my confidence. I feel like I've finally found the perfect, versatile stack for all my projects instead of trying out a new one each time.",
+  },
+  {
+    name: 'Tim Skaggs',
+    role: 'Founder @ Antler US',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1802196804236091392/ZG0OE_fO_400x400.jpg',
+    socialUrl: 'https://twitter.com/tskaggs',
+    quote: 'Nearly done with a MVP in 3 days of part-time work... and deployed on Fly.io in 10 minutes.',
+  },
+  {
+    name: 'Cam Blackwood',
+    role: 'Founder @ Microinfluencer.club',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1927721707164377089/it8oCAkf_400x400.jpg',
+    socialUrl: 'https://twitter.com/CamBlackwood95',
+    quote: 'Setting up a full stack SaaS in 1 minute with WaspLang.',
+  },
+  {
+    name: 'JLegendz',
+    role: 'Enterpreneur',
+    avatarSrc:
+      'https://cdn.discordapp.com/avatars/1003468772251811921/a_c6124fcbee5621d1ad9cca83a102c4a9.png?size=80',
+    socialUrl: 'https://discord.com/channels/686873244791210014/1080864617347162122/1246388561020850188',
+    quote:
+      "Just randomly wanted to say that I've been loving working with Wasp so far. The open-saas template is great starting point and great way to learn how Wasp works. The documentation is superb and I see the GitHub is super active. The team is super responsive and the ai kapa rocks! So thanks for the work you all are doing. I’ve done plenty of with  react in the past but I’m a front end person.  With wasp though, I'm managing my db, back end functions, actions, queries, all with so much ease. I occasionally get stuck on an issue but within a day or two, and thanks to a couple of AI assistants, I get through it. So thank you!",
+  },
+  {
+    name: 'Dimitrios Mastrogiannis',
+    role: 'Founder @ Kivo.dev',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1771240035020324864/bcNSr-dA_400x400.jpg',
+    socialUrl: 'https://twitter.com/dmastroyiannis',
+    quote: "Without Wasp & Open SaaS, Kivo.dev wouldn't exist",
+  },
+  {
+    name: 'Alex Ionascu',
+    role: 'Entrepreneur',
+    avatarSrc: 'https://pbs.twimg.com/profile_images/1356710335001018376/HEfgRu8X_400x400.jpg',
+    socialUrl: 'https://twitter.com/alexandrionascu',
+    quote: "Wasp is like hot water. You don't realise how much you need it unless you try it.",
+  },
+  {
+    name: 'Emm Ajayi',
+    role: 'Enterpreneur',
+    avatarSrc:
+      'https://media2.dev.to/dynamic/image/width=320,height=320,fit=cover,gravity=auto,format=auto/https%3A%2F%2Fdev-to-uploads.s3.amazonaws.com%2Fuploads%2Fuser%2Fprofile_image%2F1215082%2F860c57b3-ac41-420b-9420-b6efda743596.jpg',
+    socialUrl: 'https://dev.to/wasp/our-web-framework-reached-9000-stars-on-github-9000-jij#comment-2dech',
+    quote:
+      "This is exactly the framework I've been dreaming of ever since I've been waiting to fully venture into the JS Backend Dev world. I believe Wasp will go above 50k stars this year. The documentation alone gives me the confidence that this is my permanent Nodejs framework and I'm staying with Wasp. Phenomenal work by the team...  Please keep up your amazing spirits. Thank you",
+  },
+];
+
+export const faqs = [
+  {
+    id: 1,
+    question: 'Why is this SaaS Template free and open-source?',
+    answer:
+      'We believe the best product is made when the community puts their heads together. We also believe a quality starting point for a web app should be free and available to everyone. Our hope is that together we can create the best SaaS template out there and bring our ideas to customers quickly.',
+  },
+  {
+    id: 2,
+    question: "What's Wasp?",
+    href: 'https://wasp-lang.dev',
+    answer:
+      "It's the fastest way to develop full-stack React + NodeJS + Prisma apps and it's what gives this template superpowers. Wasp relies on React, NodeJS, and Prisma to define web components and server queries and actions. Wasp's secret sauce is its compiler which takes the client, server code, and config file and outputs the client app, server app and deployment code, supercharging the development experience. Combined with this template, you can build a SaaS app in record time.",
+  },
+];
+export const footerNavigation = {
+  app: [
+    { name: 'Github', href: GithubUrl },
+    { name: 'Documentation', href: DocsUrl },
+    { name: 'Blog', href: BlogUrl },
+  ],
+  company: [
+    { name: 'About', href: 'https://wasp.sh' },
+    { name: 'Privacy', href: '#' },
+    { name: 'Terms of Service', href: '#' },
+  ],
+};
+
+export const examples = [
+  {
+    name: 'Example #1',
+    description: 'Describe your example here.',
+    imageSrc: openSaasBannerWebp,
+  },
+  {
+    name: 'Example #2',
+    description: 'Describe your example here.',
+    imageSrc: openSaasBannerWebp,
+  },
+  {
+    name: 'Example #3',
+    description: 'Describe your example here.',
+    imageSrc: openSaasBannerWebp,
+  },
+  {
+    name: 'Example #4',
+    description: 'Describe your example here.',
+    imageSrc: openSaasBannerWebp,
+  },
+  {
+    name: 'Example #5',
+    description: 'Describe your example here.',
+    imageSrc: openSaasBannerWebp,
+  },
+];
