--- template/app/src/landing-page/LandingPage.tsx
+++ opensaas-sh/app/src/landing-page/LandingPage.tsx
@@ -1,3 +1,4 @@
+import { Admin, AIReady, Auth, Payments } from "./components/Examples";
 import ExamplesCarousel from "./components/ExamplesCarousel";
 import FAQ from "./components/FAQ";
 import FeaturesGrid from "./components/FeaturesGrid";
@@ -11,7 +12,6 @@
   footerNavigation,
   testimonials,
 } from "./contentSections";
-import AIReady from "./ExampleHighlightedFeature";
 
 export default function LandingPage() {
   return (
@@ -19,6 +19,9 @@
       <main className="isolate">
         <Hero />
         <ExamplesCarousel examples={examples} />
+        <Auth />
+        <Payments />
+        <Admin />
         <AIReady />
         <FeaturesGrid features={features} />
         <Testimonials testimonials={testimonials} />
