--- template/app/src/server/queries.ts
+++ opensaas-sh/app/src/server/queries.ts
@@ -110,6 +110,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true
         },
         {
           OR: [
@@ -150,6 +151,7 @@
             mode: 'insensitive',
           },
           isAdmin: args.isAdmin,
+          isMockUser: true
         },
         {
           OR: [
