--- /Users/ilakovac/dev/open-saas/template-test/tools/../../template-test/base-app/app/main.wasp
+++ template-test/app/main.wasp
@@ -39,7 +39,7 @@
       email: {
         fromField: {
           name: "Open SaaS App",
-          email: "me@example.com"
+          email: "opensaastemplate@mg.wasp.sh"
         },
         emailVerification: {
           clientRoute: EmailVerificationRoute,
@@ -84,14 +84,11 @@
   },
 
   emailSender: {
-    // NOTE: "Dummy" provider is just for local development purposes.
-    //   Make sure to check the server logs for the email confirmation url (it will not be sent to an address)!
-    //   Once you are ready for production, switch to e.g. "SendGrid" or "Mailgun" providers. Check out https://docs.opensaas.sh/guides/email-sending/ .
-    provider: Dummy,
+    provider: SMTP,
     defaultFrom: {
       name: "Open SaaS App",
       // When using a real provider, e.g. SendGrid, you must use the same email address that you configured your account to send out emails with!
-      email: "me@example.com"
+      email: "opensaastemplate@mg.wasp.sh"
     },
   },
 }
