--- template/app/src/payment/stripe/paymentDetails.ts
+++ opensaas-sh/app/src/payment/stripe/paymentDetails.ts
@@ -19,7 +19,7 @@
 ) {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       datePaid,
@@ -46,7 +46,7 @@
 ) {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       subscriptionPlan: paymentPlanId,
