--- template/app/src/payment/stripe/paymentDetails.ts
+++ opensaas-sh/app/src/payment/stripe/paymentDetails.ts
@@ -14,10 +14,10 @@
 ) => {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: userStripeId
+      stripeId: userStripeId
     },
     data: {
-      paymentProcessorUserId: userStripeId,
+      stripeId: userStripeId,
       subscriptionPlan,
       subscriptionStatus,
       datePaid,
