--- template/app/src/payment/stripe/paymentDetails.ts
+++ opensaas-sh/app/src/payment/stripe/paymentDetails.ts
@@ -26,7 +26,7 @@
 ) {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       datePaid,
@@ -48,7 +48,7 @@
 ) {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       subscriptionPlan: paymentPlanId,
