--- template/app/src/payment/stripe/paymentProcessor.ts
+++ opensaas-sh/app/src/payment/stripe/paymentProcessor.ts
@@ -13,8 +13,8 @@
 } from "./checkoutUtils";
 import { stripeClient } from "./stripeClient";
 import {
-  fetchUserPaymentProcessorUserId,
-  updateUserPaymentProcessorUserId,
+  fetchUserStripeId,
+  updateUserStripeId
 } from "./user";
 import { stripeMiddlewareConfigFn, stripeWebhook } from "./webhook";
 
@@ -28,10 +28,10 @@
   }: CreateCheckoutSessionArgs) => {
     const customer = await ensureStripeCustomer(userEmail);
 
-    await updateUserPaymentProcessorUserId(
+    await updateUserStripeId(
       {
         userId,
-        paymentProcessorUserId: customer.id,
+        stripeId: customer.id,
       },
       prismaUserDelegate,
     );
@@ -59,19 +59,19 @@
     prismaUserDelegate,
     userId,
   }: FetchCustomerPortalUrlArgs) => {
-    const paymentProcessorUserId = await fetchUserPaymentProcessorUserId(
+    const stripeId = await fetchUserStripeId(
       userId,
       prismaUserDelegate,
     );
 
-    if (!paymentProcessorUserId) {
+    if (!stripeId) {
       return null;
     }
 
     const billingPortalSession =
       await stripeClient.billingPortal.sessions.create({
-        customer: paymentProcessorUserId,
-        return_url: `${env.WASP_WEB_CLIENT_URL}account`,
+        customer: stripeId,
+        return_url: `${env.WASP_WEB_CLIENT_URL}/account`,
       });
 
     return billingPortalSession.url;
