--- template/app/src/payment/stripe/user.ts
+++ opensaas-sh/app/src/payment/stripe/user.ts
@@ -4,7 +4,7 @@
 import type { SubscriptionStatus } from "../plans";
 import { PaymentPlanId } from "../plans";
 
-export async function fetchUserPaymentProcessorUserId(
+export async function fetchUserStripeId(
   userId: User["id"],
   prismaUserDelegate: PrismaClient["user"],
 ): Promise<string | null> {
@@ -13,20 +13,20 @@
       id: userId,
     },
     select: {
-      paymentProcessorUserId: true,
+      stripeId: true,
     },
   });
 
-  return user.paymentProcessorUserId;
+  return user.stripeId;
 }
 
-interface UpdateUserPaymentProcessorUserIdArgs {
+interface UpdateUserStripeIdArgs {
   userId: User["id"];
-  paymentProcessorUserId: NonNullable<User["paymentProcessorUserId"]>;
+  stripeId: NonNullable<User["stripeId"]>;
 }
 
-export function updateUserPaymentProcessorUserId(
-  { userId, paymentProcessorUserId }: UpdateUserPaymentProcessorUserIdArgs,
+export function updateUserStripeId(
+  { userId, stripeId }: UpdateUserStripeIdArgs,
   prismaUserDelegate: PrismaClient["user"],
 ): Promise<User> {
   return prismaUserDelegate.update({
@@ -34,7 +34,7 @@
       id: userId,
     },
     data: {
-      paymentProcessorUserId,
+      stripeId
     },
   });
 }
@@ -55,7 +55,7 @@
 ): Promise<User> {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       datePaid,
@@ -82,7 +82,7 @@
 ): Promise<User> {
   return userDelegate.update({
     where: {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
     },
     data: {
       subscriptionPlan: paymentPlanId,
