--- template/app/src/payment/stripe/webhook.ts
+++ opensaas-sh/app/src/payment/stripe/webhook.ts
@@ -110,7 +110,7 @@
     case PaymentPlanId.Credits10:
       await updateUserCredits(
         {
-          paymentProcessorUserId: customerId,
+          stripeId: customerId,
           datePaid: invoicePaidAtDate,
           numOfCreditsPurchased: paymentPlans[paymentPlanId].effect.amount,
         },
@@ -121,7 +121,7 @@
     case PaymentPlanId.Hobby:
       await updateUserSubscription(
         {
-          paymentProcessorUserId: customerId,
+          stripeId: customerId,
           datePaid: invoicePaidAtDate,
           paymentPlanId,
           subscriptionStatus: SubscriptionStatus.Active,
@@ -168,7 +168,7 @@
   );
 
   const user = await updateUserSubscription(
-    { paymentProcessorUserId: customerId, paymentPlanId, subscriptionStatus },
+    { stripeId: customerId, paymentPlanId, subscriptionStatus },
     prismaUserDelegate,
   );
 
@@ -236,7 +236,7 @@
 
   await updateUserSubscription(
     {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
       subscriptionStatus: SubscriptionStatus.Deleted,
     },
     prismaUserDelegate,
