--- template/app/src/payment/stripe/webhook.ts
+++ opensaas-sh/app/src/payment/stripe/webhook.ts
@@ -111,7 +111,7 @@
     case PaymentPlanId.Credits10:
       await updateUserCredits(
         {
-          paymentProcessorUserId: customerId,
+          stripeId: customerId,
           datePaid: invoicePaidAtDate,
           numOfCreditsPurchased: paymentPlans[paymentPlanId].effect.amount,
         },
@@ -122,7 +122,7 @@
     case PaymentPlanId.Hobby:
       await updateUserSubscription(
         {
-          paymentProcessorUserId: customerId,
+          stripeId: customerId,
           datePaid: invoicePaidAtDate,
           paymentPlanId,
           subscriptionStatus: SubscriptionStatus.Active,
@@ -169,7 +169,7 @@
   );
 
   const user = await updateUserSubscription(
-    { paymentProcessorUserId: customerId, paymentPlanId, subscriptionStatus },
+    { stripeId: customerId, paymentPlanId, subscriptionStatus },
     prismaUserDelegate,
   );
 
@@ -237,7 +237,7 @@
 
   await updateUserSubscription(
     {
-      paymentProcessorUserId: customerId,
+      stripeId: customerId,
       subscriptionStatus: SubscriptionStatus.Deleted,
     },
     prismaUserDelegate,
